src/gui/widgets/settings.cpp

```diff
@@ -478,7 +478,7 @@ static std::map<std::string, gui_definition> guis;
 static std::map<std::string, gui_definition>::const_iterator current_gui = guis.end();
 
 /** Points to the default gui. */
-static std::map<std::string, gui_definition>::const_iterator default_gui = guis.end();
+static std::map<std::string, gui_definition>::iterator default_gui = guis.end();
```

Comment:
You should make `current_gui` non-const instead of `default_gui`. (They'll actually refer to the same thing unless someone decides to start retheming GUI2.)